### DISCUSSION TITLE
Missing patient ID error

### DISCUSSION THREAD

**Discussion by sbabu2022** [creepymonster/GlucoseDirect#426] [General]
I am attempting to activate a Libre 3 sensor with libre linkup connection. Getting an error, “Missing patient ID”. Should I be not scanning the sensor? What am I missing? Thank you for your help in advance.

**Comment by creepymonster** [comment]
Hello,

the Missing PatientID message usually appears when the LibreLinkUp login was not completed.

Only when LibreLinkUp receives data, GD can do that too.

Probably the LibreView account was not entered in the Libre 3 app under connected apps > LibreLinkUp. Or afterwards the invitation was not accepted in the LibreLinkUp App.

Here is a video that shows the whole process.

Connect "Libre 3" App with "LibreLinkUp".
https://youtube.com/shorts/Ljj-eQ4Jl30

Many greetings
Reimar
